src/opentype/woff.js

```diff
@@ -1,7 +1,8 @@
 import { SimpleTable } from "./tables/simple-table.js";
 import lazy from "../lazy.js";
+import { _window } from "../../lib/window.js";
 
-const gzipDecode = (window.pako ? window.pako.inflate : undefined);
+const gzipDecode = (_window.pako ? _window.pako.inflate : undefined);
```

Comment:
This feels like it would break loading of WOFF files?

Reply:
No, both pako and unbrotli are correctly assigned to the global variable, in Node. That is why we use the _window override.

Reply:
That... should never happen? The global is effectively off limits in node, you use `require` (which is a cached dependency loader) for this kind of thing.